presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -45,10 +45,10 @@
     private final PlanNode root;
     private final Map<Symbol, Type> symbols;
     private final PartitioningHandle partitioning;
-    private final List<PlanNodeId> partitionedSources;
-    private final Set<PlanNodeId> partitionedSourcesSet;
+    private final List<PlanNodeId> tableScanSources;
+    private final Set<PlanNodeId> tableScanSourcesSet;
```

Comment:
This field doesn't belong here. It is used only in the `HttpRemoteTask`. Let's keep that set out there.

Reply:
@arhimondr : It is also weird for `HttpRemoteTask` to hold a set of `PlanNode`. So I will keep it as is for now.